app/kumactl/cmd/install/install_dns.go

```diff
@@ -0,0 +1,92 @@
+package install
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const appendTemplate = `kuma:53 {
+        errors
+        cache 30
+        %s . %s:5653
+    }`
+
+var resourceHeader = []byte("apiVersion: v1\nkind: ConfigMap\n")
+
+func newInstallDNS() *cobra.Command {
```

Comment:
We have tests for every `kumactl install ...` command. Could you please add one for this

Reply:
The test will be part of the E2E, you can not really test it without a valid K8s controller. There is a sentence about this at the end of the PR description.